### ISSUE TITLE
No Reboot on App Install

### ISSUE THREAD

**Issue by InsaneSplash** [mynodebtc/mynode#741]
Is it possible to not reboot on every app installation?

Evertime a change is made, a reboot is required which doesn't seem to make sense. For example... you make a change to lnd.conf, all that is needed is to restart the lnd process, not reboot the operating system.

Example: Installing  lntop does not need a reboot?

- [X ] Mark this if you searched the existing list of open/closed issue and did not find anything similar

**Comment by tehelsper** [comment]
Yeah, that would be nice. The history here is that early on the Raspi3 and first Raspi4 didn't have much RAM so to install apps, everything needed to shutdown while they were being built and installed or the install could fail due to lack of available RAM. Some other apps didn't handle LND and BTC restarts without them also restarting and so the process became: stop services, install new stuff, reboot.

This isn't as necessary any longer and is definitely possible for some of the new apps being added. I'll investigate adding a way for some apps to install without a full reboot.

**Comment by InsaneSplash** [comment]
That would really help. What I ended up doing is updating the versions of some of the apps I wanted to update, then I update 1 of them, and all would then be updated rather than doing it individually. I do think that this would help keep a much better uptime on the node.
